specification/maps/data-plane/Route/preview/2023-10-01-preview/route.json

```diff
@@ -278,7 +278,7 @@
               {
                 "value": "fastestAvoidClosureWithoutTraffic",
                 "name": "FastestAvoidClosureWithoutTraffic",
-                "description": "The route is calculated to minimize the time and avoid road closures. No traffic information except for road closures is used in the calculation."
+                "description": "The route is calculated to minimize the time and avoid road closures. No traffic information except for road closures is used in the calculation. `Note`: Only supported for driving and walking travelMode."
```

Comment:
@koyasu221b Can you please check if Bing routing gives a response for walking for fastestAvoidClosureWithoutTraffic